index.html

```diff
@@ -8,41 +9,343 @@
+            <div class="holder-title">
+                <h5>Multi-Post Stories Gain+Glory</h5>
+            </div>
+            <div class="holder-discription">
+
+                <ul class="tag">
+
+                    <li class="tag-item" >
+                        Ruby on rails
+                    </li>
+                    <li class="tag-item" >
+                        Css
+                    </li>
+                    <li class="tag-item" >
+                        JavaScript
+                    </li>
+                    <li class="tag-item" >
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
+
+            </div>
+        </div>
+        <div class="card-holder-item">
+
+            <div class="holder-image">
+
+            </div>
+            <div class="holder-title">
+                <h5>Multi-Post Stories Gain+Glory</h5>
+            </div>
+            <div class="holder-discription">
+
+                <ul class="tag">
+
+                    <li class="tag-item" >
+                        Ruby on rails
+                    </li>
+                    <li class="tag-item" >
+                        Css
+                    </li>
+                    <li class="tag-item" >
+                        JavaScript
+                    </li>
+                    <li class="tag-item" id="fourth-item">
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
+
+            </div>
+        </div>
+
+    </div>
+
+    <div class="about-me">
+
+        <div id="title">
+            <h2>About me</h2>
+        </div>
+        <div class="about-me-body">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.
+        </div>
+        <div id="btn-2">
+            <button type="button" class="btn-2">Get My Resume</button>
+        </div>
+    </div>
+    <div class="skills">
+        <div class="skils-container">
+
+            <div class="icon">
+                <img src="./icons/languages.svg" alt="langauges">
+            </div>
+            <div class="title">
+                <h2>Languages</h2>
+            </div>
+            <div class="discription">
+                <ul class="skills-tag">
+
+                    <li class="skillstag-item">
+                        JavaScript
+                    </li>
+                    <li class="skillstag-item">
+                        Ruby
+                    </li>
+                    <li class="skillstag-item">
+                        HTML
+                    </li>
+                    <li class="skillstag-item">
+                        CSS
+                    </li>
+                </ul>
+            </div>
+
+        </div>
```

Comment:
Your skills cards having a big space at the bottom, please try to stick to the Figma design 

- [ ] Your design 
![image](https://user-images.githubusercontent.com/61790587/164763056-2d2abf9d-196f-4838-8f0f-4fa1dbccd648.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/164763088-a9138b50-1bf5-43fd-80d3-f8dcb9028d67.png)